__docs__/wonder-blocks-modal/drawer-launcher-testing-snapshots.stories.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from "react";
+import type {Meta, StoryObj} from "@storybook/react";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import {BodyText} from "@khanacademy/wonder-blocks-typography";
+
+import {FlexibleDialog, DrawerLauncher} from "@khanacademy/wonder-blocks-modal";
+
+import type {ModalElement} from "../../packages/wonder-blocks-modal/src/util/types";
+
+import {allModes} from "../../.storybook/modes";
+import {reallyLongText} from "../components/text-for-testing";
+
+const customViewports = {
+    phone: {
+        name: "phone",
+        styles: {
+            width: "320px",
+            height: "568px",
+        },
+    },
+    tablet: {
+        name: "tablet",
+        styles: {
+            width: "640px",
+            height: "960px",
+        },
+    },
+    desktop: {
+        name: "desktop",
+        styles: {
+            width: "1024px",
+            height: "768px",
+        },
+    },
+} as const;
+
+const DefaultModal = (): ModalElement => (
+    <FlexibleDialog
+        title="Single-line title"
+        content={
+            <View>
+                <BodyText>{reallyLongText}</BodyText>
+            </View>
+        }
+    />
+);
+
+export default {
+    title: "Packages / Modal / Testing / Snapshots / DrawerLauncher",
+    component: DrawerLauncher,
+    parameters: {
+        chromatic: {
+            modes: {
+                default: allModes.themeDefault,
+                "default rtl": allModes["themeDefault rtl"],
+                small: allModes.small,
+                large: allModes.large,
```

Comment:
I think we can remove the `large` mode since the `default`, `default rtl`, and `thunderblocks` modes are all at large screen sizes too! This can reduce our num of snapshots even more!